src/index.html

```diff
@@ -1,15 +1,166 @@
+      </div>
+      <div>
+        <label for="birthday">Full date of birth:</label>
+        <input type="date" name="birthday" id="birthday">
+      </div>
+      <div>
+        <label for="agreement">I accept the term of the agreement</label>
+        <input type="checkbox" name="agreement" id="agreement">
+      </div>
+    </fieldset>
+
+    <fieldset>
+      <legend>Registration</legend>
+      <div>
+        <label for="email">E-mail:</label>
+        <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="email@example.com"
+          required
+        >
+      </div>
+      <div>
+        <label for="password">Password:</label>
+        <input
+          type="password"
+          name="password"
+          id="password"
+          minlength="8"
+          maxlength="20"
+          required
+        >
+      </div>
+    </fieldset>
+
+    <fieldset>
+      <legend>An interesting fact about you!</legend>
+      <div>
+        Do you love cats?
+        <input
+          type="radio"
+          id="catsYes"
+          name="catsYes"
+          value="yes"
+        >
+        <label for="catsYes">Yes</label>
+        <input
+          type="radio"
+          id="catsNo"
+          name="catsNo"
+          value="no"
+        >
+        <label for="catsNo">No</label>
+      </div>
+      <div>
+        <label for="color">What is your favorite color?</label>
+        <input type="color" name="color" id="color">
+      </div>
+      <div>
+        <label for="bedtime">What time do you go to bed?</label>
+        <input
+          type="time"
+          name="bedtime"
+          id="bedtime"
+          step="2"
+        >
+      </div>
+      <div>
+        <label for="cars">What are your favorite brand of cars?</label>
+        <select
+          id="cars"
+          name="cars"
+          size="4"
+          multiple
+        >
+          <option value="bmw">BMW</option>
+          <option value="audi">Audi</option>
+          <option value="Lada">Lada</option>
+        </select>
+      </div>
+      <div>
+        <label for="work-rate">How do you rate our work?</label>
+        <input
+          type="range"
+          name="work-rate"
+          id="work-rate"
+          value="0"
+        >
+      </div>
+    </fieldset>
+
+    <fieldset>
+      <legend>Additional info:</legend>
+      <div>
+        <label for="comments">Comments:</label>
+        <textarea
+          name="comments"
+          id="comments"
+          rows="2"
+          cols="24"
+        ></textarea>
+      </div>
+      <div>
+        <label for="recommendation">Would you recommend us?</label>
+        <select name="recommendation" id="recommendation">
+          <option value="yes">yes</option>
+          <option value="no">no</option>
+        </select>
+      </div>
+    </fieldset>
+
+    <button
+      type="submit"
+      name="button"
+      value="Submit"
+      onsubmit = "onSubmit()"
+    >Submit</button>
```

Comment:
```suggestion
    <button
      type="submit"
      name="button"
      value="Submit"
      onsubmit = "onSubmit()"
    >
      Submit
    </button>
```
it was mentioned in the group